tests/v2/validation/deleting/delete_cluster_rke2_test.go

```diff
@@ -0,0 +1,65 @@
+package deleting
```

Comment:
future works for this package; would be nice for the user not to have to specify the engine type to delete the cluster, just need them to specify the cluster name.

Reply:
That's a fair point. I think that could be an enhancement we look at after this is merged since it's a bit out the scope

Reply:
yeah, I'll make a ticket after this is merged